src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -197,6 +197,10 @@
 	padding: 0 22px 0 6px;
 }
 
+#terminal-tabs-plus-button {
+	padding-left: 20px;
+}
+
```

Comment:
We'll eventually want to style it similar to the buttons in the panel, I'm not sure how the hover feedback for the "New Terminal" text will work though.

Reply:
Created #120960